Library/Formula/coreutils.rb

```diff
@@ -35,8 +37,13 @@ def install
       --program-prefix=g
     ]
     args << "--without-gmp" if build.without? "gmp"
-    system "./configure", *args
-    system "make", "install"
+
+    src = Pathname.pwd
+    mktemp do
+      system "#{src}/configure", *args
+      system "make", "install"
+      install_dsym if build.dsym?
```

Comment:
All the object files would be retained in the build directory, which is retained in KEG/Source